src/components/messages/Message/Message.scss

```diff
@@ -1,5 +1,51 @@
+$bounce-curve: cubic-bezier(0.34, 1.56, 0.64, 1);
+
 .Message {
+  $parent: &;
+
   display: flex;
+  position: relative;
+  &--recent {
+    animation: bounce-up 0.2s $bounce-curve forwards;
+  }
+  &__container {
+    &--first {
```

Comment:
This looks a bit overkill and doesn't match the behavior designed in Figma, please take a look at [this frame](https://www.figma.com/file/P70QpTDpsgwdPJ2jJmVXxg/web3inbox-standalone?node-id=85%3A6093&t=PnaTzlxGXGYowXT2-4).
You'll notice that the gap between "stacked" messages is smaller and the top right and bottom right border radius are 6px.